docs/playbooks/mas.md

```diff
@@ -57,12 +57,16 @@ To utilise this feature you must set the optional `MAS_DOMAIN` detailed previous
 
 ### Required environment variables
 - `CIS_CRN` which can be obtained from your CIS service overview page, it will be in the format: `crn:v1:bluemix:public:internet-svcs:global:a/02fd888448c1415baa2bcd65684e4db3:9969652f-6955-482b-b59c-asdasasdede50c::`
-- `IBMCLOUD_APIKEY` Your IBM Cloud API key with DNS API write access. Note: (This API key will be stored in your cluster for DNS challenge when requesting new certs)
-
-!!! important "TODO"
-    Support `CIS_APIKEY` instead as we don't really want to re-use the main `IBMCLOUD_APIKEY` API key as it has too wide scope for CIS.  Document what scope to set up the API key with and what steps etc.
-
-    Could we even use our API key to create this new API key automatically so that the user does not need to do anything?
+- `CIS_APIKEY` A Service ID API key with DNS API Editor/Manager access.  Note: (This API key will be stored in your cluster for DNS challenge when requesting new certs)
+
+The process for creating a CIS_APIKEY:
+- Create a Service ID
+- Create an "Access Policy" with
```

Comment:
@bwbudak you are not informing against what resource Access Policy must be created here. For example:

<img width="598" alt="image" src="https://user-images.githubusercontent.com/15021471/152203693-b359af0a-ccbc-41f8-8f65-67bc9ebc9ef2.png">

We need to create an access policy of Editor/Manager against "Internet Services service" (we may have other Platform/Service access among our policies, but against other resources that is not "Internet Services"... btw, I thought these were the permissions we already had when we tested, or not? They look the same.

we are saying here to create an "Access Policy" but you are not specifying which one

Reply:
Modified the wording to include the fact that it is restricting access to the **Internet Services service**